Library/Homebrew/rubocops/version.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "rubocops/extend/formula"
+
+module RuboCop
+  module Cop
+    module FormulaAudit
+      class Version < FormulaCop
+        def audit_formula(_node, _class_node, _parent_class_node, body_node)
+          version_node = find_node_method_by_name(body_node, :version)
+          return unless version_node
+
+          version = string_content(parameters(version_node).first)
+
+          problem "version is set to an empty string" if version.blank?
```

Comment:
I'm not sure if this is even needed since if the version is empty the following error message gets thrown:

```                                                                                                                                               
Error: invalid attribute for formula 'FooBar': version ("")
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/formula.rb:266:in `validate_attributes!'
/usr/local/Homebrew/Library/Homebrew/formula.rb:213:in `initialize'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:122:in `new'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:122:in `get_formula'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:332:in `factory'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:73:in `resolve'
/usr/local/Homebrew/Library/Homebrew/cli/args.rb:99:in `block in resolved_formulae'
/usr/local/Homebrew/Library/Homebrew/cli/args.rb:98:in `map'
/usr/local/Homebrew/Library/Homebrew/cli/args.rb:98:in `resolved_formulae'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:92:in `audit'
/usr/local/Homebrew/Library/Homebrew/brew.rb:110:in `<main>'
```

Or am I misunderstanding things? 

Check was migrated from https://github.com/Homebrew/brew/pull/7586/files#diff-bd0ba063e0f8aeb1f4a3997d34914c0cL1045-L1047

Reply:
Would still be nice so this shows up in `rubocop` even if Homebrew can't parse it.

Reply:
I see, thanks for the explanation.